packages/icons/src/trimet-mode-icon-2021.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Bicycle, Car, Micromobility, Walk } from "./trimet-2021";
+
+/**
+ * Icons for 2021 TriMet mode selector.
+ */
+
+function TriMetModeIcon({ mode, ...props }) {
```

Comment:
Add a story showing these new icons.

Reply:
Added in 777cc35e40bdde1c2fda0f3c0648b37b487f266d